src/com/untamedears/realisticbiomes/listener/PlayerListener.java

```diff
@@ -45,10 +49,19 @@ public void onPlayerInteractEvent(PlayerInteractEvent event) {
 			return;
 		}
 		
-		Plant plant = null;
-		
 		Block block = event.getClickedBlock();
 		
+		if (((List<ItemStack>)block.getDrops()).get(0).getType() == event.getMaterial()) {
+			suppression.put(event.getPlayer().getUniqueId(), System.currentTimeMillis());

```

Comment:
@ProgrammerDan do you think it's okay that this lis never gets cleared? It should be filling up with players over the course of the day.